README.md

```diff
@@ -1,2 +1,43 @@
-# Interstellar_Project
-This repository contains the pipelines used to analyze the data sets from the Interstellar collaboration.
+# Pyroptosis signature project 
+
+This repository contains the pipelines used to analyze the "waves" of data for the Interstellar collaboration project.
```

Comment:
Consider linking to the Interstellar collaboration project (if available).

Reply:
I have now changed it to also reference Mike's repository for the data analysis/machine learning. I do not believe there is a document for this project yet that is open source, but it will be added when available 😄 Thank you for bringing my attention to this!